README.md

```diff
@@ -1,7 +1,69 @@
-# sdk-php
+# PHP SDK for [CloudEvents](https://github.com/cloudevents/spec)
+
+## Status
+
+This SDK is currently a work in progress, therefore things might (and will) break with every update.
+
+This SDK aims to supports the following versions of CloudEvents:
+
+- [v1.0](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md)
```

Comment:
Probably want exactly v1.0: https://github.com/cloudevents/spec/blob/v1.0/spec.md

Reply:
I chose v1.0.1 specifically because there are [substantial differences](https://github.com/cloudevents/spec/compare/v1.0...cloudevents:v1.0.1) in comparison between the v1.0 and v1.0.1 branches. There are no breaking changes between v1.0 as referenced in the same way as specversion, but much more clarity in the v1.0.1 release. Both are supported as part of v1.0.1.